packages/mpv/mpv.conf

```diff
@@ -0,0 +1,10 @@
+# Default configuration for mpv
+
+# Enable OpenSL ES output, since this should work on all Android devices
+ao=opensles
+
+# Increase audio buffer to help with stuttering with bluetooth devices. See #1127.
+audio-buffer=0.5
+
+# Disable Video Output. Termux doesn't support video output (with the exception of "tct").
+vo=null
```

Comment:
You claim that this is needed to disable video output, yet this option doesn't appear in old versions. What changed to require this flag?